src/util.cpp

```diff
@@ -484,6 +484,24 @@ const boost::filesystem::path &ZC_GetParamsDir()
     return path;
 }
 
+// Return the user specified export directory.  Create directory if it doesn't exist.
+// If user did not set option, or if the path is not a directory, return an empty path.
```

Comment:
Given that the desired behaviour is to never export anything if `-exportdir` isn't set, I'd prefer that we throw an exception here. As-is, we are relying on the caller to check the return value, which seems prone to future breakage.